packages/toast/src/ToastContext/ToastContext.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+import { ToastContainer } from '../ToastContainer';
+
+import { ToastContextProps, ToastProviderProps } from './ToastContext.types';
+import { useToastReducer } from './ToastReducer';
+
+export const ToastContext = React.createContext<ToastContextProps>({
+  pushToast: () => '',
+  popToast: () => undefined,
+  updateToast: () => undefined,
+  getToast: () => undefined,
+  getStack: () => undefined,
+  clearStack: () => {},
+});
+
+export const ToastProvider = ({
+  children,
+  initialValue,
+}: React.PropsWithChildren<ToastProviderProps>) => {
+  const { stack, pushToast, popToast, updateToast, getToast, clearStack } =
+    useToastReducer(initialValue);
+
+  return (
+    <ToastContext.Provider
+      value={{
+        pushToast,
+        popToast,
+        updateToast,
+        getToast,
+        getStack: () => stack,
```

Comment:
this might be a dumb q, but could it not just pass `stack` instead of a `getStack` function?

Reply:
Since we don't necessarily want users to access the stack itself and manipulate it directly